src/main.test.js

```diff
@@ -270,6 +269,11 @@ test('Zips Go function files', async t => {
     const { stdout } = await execa(unzippedFile)
     t.is(stdout, 'test')
   }
+
+  const tcFile = `${tmpDir}/netlify-toolchain`
+  await pathExists(tcFile)
```

Comment:
```suggestion
  t.true(await pathExists(tcFile))
```

Otherwise, this statement is a noop.